docs/Perlin & Elevation Prop

```diff
@@ -0,0 +1,32 @@
+Eric Meissner
+Patrick Sharp 
+
+Map and Sequence Project Proposal
+Perlin noise & Elevation
+
+For our project, we propose to implement a generic Perlin Noise function and a function to generate elevations of voxels. 
+
+Perlin Noise:
+  The Perlin Noise function is a pseudorandom function that generates smoothly scaling randomness, with points near each other having similar values while points far apart have distinct, non-correlated values. It will take in between two and four “coordinates” (x, y, z, t) and return a single floating point number. The user of the function then maps that value to any number of things, from smoothly scaled elevation maps to fog generation. We demonstrate one use in our elevation function. 
+  http://upload.wikimedia.org/wikipedia/commons/d/da/Perlin_noise.jpg (2D perlin noise mapping values between 0-1 to grayscale)
+
+The Elevation:
+  The elevation will be a public member function of SomeClass that takes in two coordinate points, x and y, and uses two private member variables (minElevation and maxElevation) and returns the z coordinate, representing the height of solid ground at that point. It will be called from the world generation function, not provided by us, only once per (x,y) coordinate point.
+  The z coordinate is generated by the Perlin noise function, and so will only generate rolling hill type landscapes for now. In the future, this can be extended to include mountains and cliffs. The elevation function calls the Perlin noise function on the world map.
+  We need to discuss exactly which class will own this function and it’s needed member variables with the “Control Guys”.
+
```

Comment:
>You could just have all these Perlin noise functions in a namespace. That way you wouldn't need to rely on someone else specifying a class for you.

I agree with this for the Perlin noise. However I believe that the elevation methods are a seperate matter.